src/wp-admin/includes/ajax-actions.php

```diff
@@ -3975,18 +3975,34 @@ function wp_ajax_crop_image() {
 
 			$size       = wp_getimagesize( $cropped );
 			$image_type = ( $size ) ? $size['mime'] : 'image/jpeg';
+			/** @var WP_Post $original_attachment */
+			$original_attachment = get_post( $attachment_id );
+			$has_title           = mb_strlen( trim( $original_attachment->post_title ) ) > 0;
+			$has_description     = mb_strlen( trim( $original_attachment->post_content ) ) > 0;
```

Comment:
A simplification (see it in action https://3v4l.org/hCPCX):

```php
( '' !== trim( $original_attachment->post_title ) );
```

```suggestion
			$has_title           = ( '' !== trim( $original_attachment->post_title ) );
			$has_description     = ( '' !== trim( $original_attachment->post_content ) );
```

Reply:
I absolutely agree. It's much smarter to compare values with an empty string.
Fixed in a2e8c8690e82677dfeaea998b027b10f72e6cbdd